Dockerfile

```diff
@@ -0,0 +1,29 @@
+# syntax=docker/dockerfile:1.0.0-experimental
+
+FROM node:10.16-buster AS builder
+
+WORKDIR /tmp
+
+COPY . .
+
+RUN mkdir -p /root/.ssh \
+&&  ssh-keyscan github.com >> ~/.ssh/known_hosts
+
+RUN --mount=type=ssh yarn install \
+&&  yarn build:static-site
+
+FROM nginx:1.17
+
+COPY --from=builder /tmp/dist/ui.lumapps.com/ /var/www
+COPY .docker/default.conf /etc/nginx/conf.d/default.conf
+COPY .docker/nginx.conf /etc/nginx/nginx.conf
+
+WORKDIR /var/www
+
+RUN touch /var/run/nginx.pid \
+&&  chown -R www-data:www-data /var/run/nginx.pid \
+&&  chown -R www-data:www-data /var/cache/nginx \
+&&  chown -R www-data:www-data /var/www
```

Comment:
I'd rather do a `chmod -R 644 && chown -R root:www-data` to prevent nginx from writing the files it serves in case of a security flaw. We're only serving static contents anyways.